### PR TITLE
Mhl fellowship kitchensink

### PR DESCRIPTION
**Pull request by MrishoLukamba**
This PR tries to set fellowship like set up like that on Kusama @shawntabrizi ,

Problem is origin macro doesnt implement OriginTrait trait on Fellowship custom Origins, Help needed.

### CODE REVIEW

File: frame/ranked-collective/src/lib.rs [hunk 1]
```diff
@@ -635,6 +637,19 @@ pub mod pallet {
 				pays_fee: Pays::No,
 			})
 		}
+
+		// Only root account can add a member to
+		#[pallet::weight(100)]
+		pub fn add_member_to_rank(
```
Comment (ggwpez) [inline frame/ranked-collective/src/lib.rs:643]: There are unrelated changes in this MR.  
Reviews go a lot faster if you tidy it up a bit first to make it minimal.

Comment (MrishoLukamba) [reply frame/ranked-collective/src/lib.rs:643]: Am pushing another PR which is clean and small for specific issue. These PR's commits got messed up

**Comment by MrishoLukamba** [comment]
This is outdated
